app/src/main/java/cm/aptoide/pt/app/AppViewManager.java

```diff
@@ -262,7 +263,7 @@ public Completable downloadApp(WalletApp walletApp) {
             .getAction()), walletApp.getAppName(), walletApp.getPackageName(),
         walletApp.getMd5sum(), walletApp.getIcon(), walletApp.getVersionName(),
         walletApp.getVersionCode(), walletApp.getPath(), walletApp.getPathAlt(), walletApp.getObb(),
-        false, walletApp.getSize()))
+        false, walletApp.getSize(), null, null))
```

Comment:
Minor,but why adding nulls here? Shouldn't we use it the same way as we are using for the other app models? That way, if appcoins wallet one day becomes an appbundle we won't have to change anything.

Reply:
already discussed, done.